examples/training/object_detection/pascal_voc/retina_net.py

```diff
@@ -38,9 +60,38 @@
 ]
 class_mapping = dict(zip(range(len(class_ids)), class_ids))
 
+flags.DEFINE_string(
+    "weights_path",
+    "weights_{epoch:02d}.h5",
+    "Directory which will be used to store weight checkpoints.",
+)
+flags.DEFINE_string(
+    "tensorboard_path",
+    "logs",
```

Comment:
The default value here is bit weird, and should it have a relative location?

Reply:
By default it is using relative path